selfdrive/car/subaru/values.py

```diff
@@ -55,6 +55,8 @@ def __init__(self, CP):
 class SubaruFlags(IntFlag):
   SEND_INFOTAINMENT = 1
   DISABLE_EYESIGHT = 2
+
+  # Static flags
```

Comment:
I was trying to come up with something complex for this, but in substitution I recommend we at least denote which we expect to be statically defined vs. dynamically detected for clarity.